week1/life_expectancy_model.py

```
@@ -0,0 +1,33 @@
+
+    def evaluate(self, X_test, y_test):
+        y_pred = self.predict(X_test)
+        mse = mean_squared_error(y_test, y_pred)
+        mae = mean_absolute_error(y_test, y_pred)  # Added MAE
+        r2 = r2_score(y_test, y_pred)
+        return mse, mae, r2
+
+    def get_feature_importance(self):
+        feature_importance = self.model.named_steps['regressor'].feature_importances_
+        feature_names = self.model.named_steps['preprocessor'].get_feature_names_out()
+        return feature_importance, feature_names
```

Ruff

No newline at end of file

Add trailing newline

(W292)